website/static_src/package.json

```diff
@@ -95,10 +96,12 @@
   },
   "dependencies": {
```

Comment:
Based on [these changes](https://github.com/helsenorgelab/helsenorgelab.no/pull/16/files#diff-34e50530c0ef4c33ae0741716ea36fcdL1) we may be able to remove some dependencies here.